### PR TITLE
FEA-164-Add placeholder to autocomplete input

### PR DESCRIPTION
**Pull request by AgataLudwiczynska-CC**

### CODE REVIEW

**Review by lukasz-konatowski-cc — APPROVED** [review]

**Review by darya-karpovich-cc — APPROVED** [review]

**Review by adrian-konopnicki-CC — APPROVED** [review]

**Review by klaudia-blazyczek-cc — APPROVED** [review]

File: src/pages/Home/components/AutocompleteInput/AutocompleteInput.tsx [hunk 1]
```diff
@@ -30,16 +30,16 @@ const AutocompleteInput = ({ onChange, value }: Props) => {
       <Select
         options={data}
         isLoading={isLoading}
-        onInputChange={value => handleInputChange(value)}
+        onInputChange={(value) => handleInputChange(value)}
```
Comment (kamil-borecki-cc) [inline src/pages/Home/components/AutocompleteInput/AutocompleteInput.tsx]: **question** why this change was made?

Comment (AgataLudwiczynska-CC) [reply src/pages/Home/components/AutocompleteInput/AutocompleteInput.tsx]: When I saved this file it happened automatically. I think it's a prettier

Comment (darya-karpovich-cc) [reply src/pages/Home/components/AutocompleteInput/AutocompleteInput.tsx]: **question** maybe it's better not to create a function  and replace it with `onInputChange={handleInputChange}`?

**Review by kamil-borecki-cc — APPROVED** [review]
